py_june/settings.py

```diff
@@ -131,3 +133,5 @@
 
 DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
 AUTH_USER_MODEL = 'users.ForumUser'
+CRISPY_ALLOWED_TEMPLATE_PACKS = "bootstrap4"
```

Comment:
lets keep one style with single quote
```suggestion
CRISPY_ALLOWED_TEMPLATE_PACKS = 'bootstrap4'
```